src/schema/types/mutation.graphql

```diff
@@ -1,4 +1,4 @@
 # This is an empty type, meant to be extended by other entities.
 type Mutation {
-  _empty: String
+    createuser(username: String!,firstName: String!,lastName: String!,password: String!): User
```

Comment:
Let's leave a space after `,`

Reply:
Mati, the idea of having different files is that you can compose mutations, types, and queries. So, the `mutation.graphql` would have only an `_empty` field (as query.graphql), and then you can create different files with more mutations, queries, and types. In this case, you could create a `user.graphql` file with everything you need for your user (fields, queries, and mutations):

user.graphql
```graphql
type User {
...
}

type Query {
  users: [User]
  user(...): User
}

type Mutation {
  createUser(...): User
}
```

These files are merged into a single file so, for example, the type `Mutation` will have all the mutations that you define in your files.

Let me know if I'm clear enough or we can hop on a call.

Reply:
I think I got it, it's done.